draft-ietf-moq-transport.md

```diff
@@ -1163,6 +1163,91 @@ has accepted a PUBLISH_NAMESPACE with a namespace that exactly matches the
 namespace for that track, it SHOULD only request it from the senders of those
 PUBLISH_NAMESPACE messages.
 
+## Discovery Track
+
+Every namespace has an associated discovery track with the Track Name
+`.discovery`.  A subscriber can subscribe to this track using `SUBSCRIBE` to
```

Comment:
We need to reserve this name (or all names that start with .?) to prevent applications from using this.